plasmapy/analysis/nullpoint.py

```diff
@@ -42,8 +42,6 @@ class NullPointError(Exception):
        change in future releases.
     """
 
-    pass
-
```

Comment:
The `pass` statement isn't necessary because the docstring is effectively a command.